.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit
+    rev: v2.1.1
+    hooks:
+    -   id: validate_manifest
+
+-   repo: local
+    hooks:
+    -   id: shellcheck
+        name: shellcheck
+        language: docker_image
+        entry: koalaman/shellcheck:stable
```

Comment:
I'm not sure how this fits together. Why are there three different versions being specified in the various files?